tests/unit/core/test_tracking.js

```diff
@@ -438,4 +440,70 @@ describe(__filename, () => {
       expect(DISCO_NAVIGATION_CATEGORY).toEqual('Discovery Navigation');
     });
   });
+
+  describe('sendBeacon', () => {
+    it('should send a becaon if navigator.sendBeacon exists', () => {
```

Comment:
s/becaon/beacon/

Reply:
And repeat for the rest of the tests (though I see now it's just copied from the earlier tests in the other file)

Reply:
I'm shocked I never typed it as `bacon`. 😉